typescript-getting-started/functions/package.json

```diff
@@ -11,7 +11,7 @@
     "typescript": "^2.6.1"
   },
   "scripts": {
-    "build": "./node_modules/.bin/tslint -p tslint.json && ./node_modules/.bin/tsc",
+    "build": "./node_modules/.bin/tslint --project tsconfig.json && ./node_modules/.bin/tsc",
```

Comment:
Actually it looks like this should be a directory so maybe:

```
"build": "./node_modules/.bin/tslint --project ./ && ./node_modules/.bin/tsc",
```

Reply:
@nicolasgarnier Yeah, the description of the cli parameter made me wonder about dir vs. file as well... I think they support both, most of the samples from their docs specify the file, so in the end I went for specifying the file instead of the dir. For example:
https://palantir.github.io/tslint/usage/type-checking/#cli
https://github.com/palantir/tslint/blame/master/src/tslint-cli.ts#L175

I'd go with specifying the file instead of the directory - what do you think?